models/restful-model-collection.coffee

```diff
@@ -63,6 +63,19 @@ class RestfulModelCollection
       callback(err) if callback
       Promise.reject(err)
 
+  expand: (id, callback = null) ->
```

Comment:
Could you define this method in `thread.cs` instead of in `restful-model-collection.coffee`? It makes little sense to call it on anything else than a thread.